src/test/groovy/unit/org/pih/warehouse/api/ProductApiTest.groovy

```diff
@@ -0,0 +1,44 @@
+package unit.org.pih.warehouse.api
+
+import io.restassured.path.json.JsonPath
+import io.restassured.specification.RequestSpecification
+import org.testng.Assert
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static io.restassured.RestAssured.*;
+import io.restassured.response.Response;
```

Comment:
1. remove `;` (in all lines above also)
2. make an indent between `import` and `class`